### PR TITLE
feature: ZENKO-1420 update bucket processor as data mover client

### PR DESCRIPTION
**Pull request by jonathan-gramain**
Modify lifecycle bucket processor to generate action messages to the
data mover and forward responses to the lifecycle-object-tasks-topic.

### CODE REVIEW

File: extensions/lifecycle/tasks/LifecycleTask.js [hunk 1]
```diff
@@ -749,48 +755,46 @@ class LifecycleTask extends BackbeatTask {
...
+                  .setAttribute('details.lastModified', obj.LastModified);
+            this._sendObjectAction(entry, err => {
                 if (!err) {
-                    log.debug('sent object entry for consumption', {
-                        method: 'LifecycleTask._compareObject',
-                        entry,
-                    });
+                    log.debug(
+                        'sent object entry for consumption', Object.assign({
+                            method: 'LifecycleTask._compareObject',
```
Comment (philipyoo) [inline extensions/lifecycle/tasks/LifecycleTask.js]: oops, sorry I probably forgot to change this before: ~~_compareObject~~  `_checkAndApplyExpirationRule`

File: extensions/lifecycle/tasks/LifecycleTask.js [hunk 2]
```diff
@@ -749,48 +755,46 @@ class LifecycleTask extends BackbeatTask {
...
```
Comment (philipyoo) [inline extensions/lifecycle/tasks/LifecycleTask.js]: same here

File: extensions/lifecycle/tasks/LifecycleTask.js [hunk 3]
```diff
@@ -939,22 +954,22 @@ class LifecycleTask extends BackbeatTask {
...
+            this._sendObjectAction(entry, err => {
                 if (!err) {
-                    log.debug('sent object entry for ' +
-                    'consumption', {
-                        method: 'LifecycleTask._compareVersion',
-                        entry,
-                    });
+                    log.debug('sent object entry for consumption',
+                        Object.assign({
+                            method: 'LifecycleTask._compareVersion',
```
Comment (philipyoo) [inline extensions/lifecycle/tasks/LifecycleTask.js]: ~~_compareVersion~~ `_checkAndApplyNCVExpirationRule`

**Review by philipyoo — APPROVED** [review]
lgtm
sorry for the log mistakes x.x

**Review by bennettbuchanan — APPROVED** [review]

**Comment by jonathan-gramain** [comment]
@bert-e approve

**Pull request merged**